.github/pull_request_template.md

```diff
@@ -17,3 +17,4 @@ This PR addresses ...
 - [ ] ran regression tests, post a link to the Jenkins job below.
       [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
 - [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)
+- [ ] All comments are resolved
```

Comment:
I would suggest moving this up in the list, because the ticket updates are the last thing to be done in the flow. How 'bout putting it between regression tests and ticket resolution.

Also, is there some convenient and/or clever way that folks can click on something to indicate when one of the items is not relevant or needed? Some folks go to the effort to strike through the text, but that requires special editing skills.

Reply:
Thanks! I moved it to where you recommended and added a note about irrelevant items. I left this as rendered text (instead of a comment) so that the link would work.

Reply:
Thanks! I spent a bit of time looking at this and found a few dead ends :(

It does not appear that multiple checkbox work in a bullet:
- [ ] rendered as a checkbox, not rendered as a checkbox [ ]

There have been requests for "indeterminate" state but so far it does not appear to be supported:
https://github.com/orgs/community/discussions/19199

I also looked at using a table but that does not produce checkable boxes (although you can manually check them by editing the markdown).

One option would be to use multiple templates although I think this would be unwieldy given the combinations of options.

Perhaps we defer the na item issue for another PR? It seems unexpectedly complicated.

Reply:
Agree to defer